src/HomeKit/HMEnums.cs

```diff
@@ -7,8 +7,8 @@ namespace HomeKit {
 	[iOS (8,0)]
 	[TV (10,0)]
 	[Native]
-	public enum HMError : nint {
 		[Watch (4,2), TV (11,2), iOS (11,2)]
+	public enum HMError : long {
```

Comment:
You're moving the `[Watch (4,2), TV (11,2), iOS (11,2)]` line from the `UnexpetedError` value to the enum itself. Typo?

Reply:
Yes, unfortunately running pmcs and filtering via grepdiff to make these changes is less than 100% accurate. I scanned for mistakes, but missed these two.